packages/reason-relay/src/ReasonRelay.re

```diff
@@ -431,6 +431,11 @@ module Network = {
   type t;
 
   type operation = {
+    /** This will be available only when using persisted queries. 
+     * And id field will be of presisted query Id. 
+     * In other cases, it will be null */
+    id: string,
+    /** When using persisted queries, text field will be null. */
```

Comment:
You can go ahead and remove this comment in the implementation file. It's already in the `rei` file and that should be enough.

Reply:
I moved the comments to rei file. I don't get the rebase part. The command says I'm already up-to date

Reply:
Thanks! Yeah that's weird, there still seems to be conflicts to resolve. Maybe try and resolve them in the web UI?

Reply:
I think it worked now.